docs/milestone-1.md

```diff
@@ -0,0 +1,169 @@
+# Milestone 1 - kintsugi testnet
+
+This initial milestone provides simple middleware logic with minimal consensus client changes, simple networking, no validator authentication, and manual safety mechanism
+
+## Architecture
+
+### Block Proposal
+
+```sequence
```

Comment:
For some reason, the sequence diagram doesn't show up under the view page

Reply:
GitHub doesn't support it unfortunately, I'll need to put the image

Reply:
Still missing the image :)

Reply:
https://github.blog/2022-02-14-include-diagrams-markdown-files-mermaid/